src/compiler/diagnosticMessages.json

```diff
@@ -1007,7 +1007,6 @@
         "category": "Error",
         "code": 1349
     },
-

```

Comment:
I think having these empty lines are just better to navigate the file. Like moving cursor between the empty line. Not surely know these lines has to be removed.